### PR TITLE
Wallet reset

### PR DESCRIPTION
**Pull request by makeasnek**
Add instructions for:

- Fixing wallet with corrupt DB
- Migrating wallet between machines

### CODE REVIEW

File: wiki/faq.md [hunk 1]
```diff
@@ -473,13 +473,25 @@ But that is a little longer
...
 
 The wallet has an automated backup in place that will make backups every 24 hours into the walletbackups folder inside of your data folder.
 
+---
+### How can I move my Gridcoin wallet to another machine?
+
+Note: These steps should work for any OS and for cross-OS migration. You should never run the same wallet on two machines at the same time because it looks to the network like an attack and your node may be banned. Additionally, you may lose coins and addresses as the wallets diverge over time. Don't do it.
+
+1. If you crunch BOINC to earn GRC, install BOINC on your new machine and attach to all your projects first. You must attach to all the same projects as before, you can set a project as "no new tasks" if you don't want to crunch it.
+2. Fully shut down the Gridcoin wallet, make sure it is not running in the system tray.
```
Comment (jamescowens) [inline wiki/faq.md:489]: shutdown the Gridcoin wallet on the source machine,

File: wiki/faq.md [hunk 2]
```diff
@@ -473,13 +473,25 @@ But that is a little longer
...
+3. Install the Gridcoin wallet on the new machine, fully shut it down after installation. You do not need to fully sync this wallet yet.
+4. Rename the wallet directory on the new machine to GridcoinResearch_old and copy the contents of your wallet directory from the original machine to the new machine in the same place
```
Comment (jamescowens) [inline wiki/faq.md:491]: You might want to remind folks where the data directory is.

File: wiki/faq.md [hunk 3]
```diff
@@ -616,6 +628,16 @@ Read the how to help section on the [testnet wiki page](testnet "wikilink")
 Try running the client with `-rescan` or run the console command
 `repairwallet`.
 
+---
+### I get the error that the wallet is corrupt, how can I fix it? I get the error "error loading blkindex.dat"
```
Comment (jamescowens) [inline wiki/faq.md:632]: We need to make sure that it is clear that wallet.dat errors are different than blkindex.dat (blockchain database) errors.

**Review by jamescowens — COMMENTED** [review]
See comments.

File: wiki/faq.md (hunk 1 above)
Comment (makeasnek) [reply wiki/faq.md]: Fixed

File: wiki/faq.md (hunk 2 above)
Comment (makeasnek) [reply wiki/faq.md:491]: Fixed

File: wiki/faq.md (hunk 3 above)
Comment (makeasnek) [reply wiki/faq.md]: Fixed

**Review by jamescowens — APPROVED** [review]
ACK

**Pull request merged**